src/Arcus.EventGrid.Proxy.Api/Arcus.EventGrid.Proxy.Api.csproj

```diff
@@ -24,10 +24,9 @@
     <PackageReference Include="Arcus.EventGrid.Publishing" Version="3.1.0" />
     <PackageReference Include="Arcus.WebApi.Correlation" Version="0.4.0" />
     <PackageReference Include="Arcus.WebApi.Logging" Version="1.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Should we remove this one?

Reply:
I don't see it used and the project uses .NET Core 3.1 with the `Microsoft.NET.Sdk.Web` SDK, so I thought not?

Reply:
if it builds it's ok